tests/test_NIRSpec.py

```diff
@@ -11,10 +11,7 @@
 from eureka.lib.readECF import MetaClass
 from eureka.lib.util import COMMON_IMPORTS, pathdirectory
 import eureka.lib.plots
-try:
-    from eureka.S2_calibrations import s2_calibrate as s2
-except ModuleNotFoundError:
-    pass
```

Comment:
We had these old lines (and the related `s2_installed` code) since S2 will fail to import if `jwst` is not installed, and we hadn't wanted to make that a mandatory installation requirement.

Reply:
I don't see any reason why someone would want to test just the NIRSpec pipeline without having `jwst` installed.  At this point, I think you and I (and GitHub) are the only ones running these tests.